src/Raven.Studio/typescript/components/common/splitView/ViewSheet.tsx

```diff
@@ -0,0 +1,132 @@
+import classNames from "classnames";
+import { ClassNameProps } from "components/models/common";
+import { useAppDispatch, useAppSelector } from "components/store";
+import { Icon } from "components/common/Icon";
+import { ComponentProps, PropsWithChildren, ReactNode, useCallback } from "react";
+import Button, { ButtonProps } from "react-bootstrap/Button";
+import { useSplitViewContext } from "./SplitView";
+import { splitViewSelectors, splitViewActions } from "./store/splitViewSlice";
+
+export type ViewSheetWidth = number | `${number}%`;
+
+export function ViewSheet(props: ComponentProps<"div"> & Required<PropsWithChildren>) {
+    const { className, children, ...rest } = props;
+
+    return (
+        <div {...rest} className={classNames("vstack", className)}>
+            {children}
+        </div>
+    );
+}
+
+interface SheetHeaderProps extends Required<PropsWithChildren>, ClassNameProps {
+    isPinHidden?: boolean;
+    isCloseHidden?: boolean;
+}
+
+function SheetHeader(props: SheetHeaderProps) {
```

Comment:
if we dont spread props argument anywhere, we should spread it in arguments. Also we can assign default value for e.g. `minPanelWidth`, `maxPanelWidth`, `initialPanelWidth`
```suggestion
function SheetHeader({isPinHidden, className, children, isCloseHidden}: SheetHeaderProps) {
```

Reply:
It doesn't really matter

<img width="716" height="205" alt="Image" src="https://github.com/user-attachments/assets/6274342b-f127-4790-a52d-2e652aee291c" />